include/openssl/boringssl_prefix_symbols.h

```diff
@@ -0,0 +1,5 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+// This file is intentionally empty, when building with the prefix enabled the
+// generated file will overwrite this one.
```

Comment:
An "overwrite" would only occur in the file placed during install after  a prefix build/installation had completed, right?  Would this statement ever be true for a consumer reading it? Perhaps the comment should be hypothetical -- "Were AWS-LC installed with symbol prefixes, this file would contain..."